### PR TITLE
Dining Stats

### PR DESCRIPTION
**Pull request by epoww**
Put old dining view and new blank statistics view in a tab bar controller (like the booking & reservations views on GSR page)

### CODE REVIEW

**Comment by joshdoman** [comment]
@epoww Can you attach a screenshot? We should also add a switch or flag so that the current home page (no tab bar) is rendered in production until the statistics view is finished.

**Comment by epoww** [comment]
Here's a screen shot of the tab bar

Simulator Screen Shot - iPhone 11 - 2020-02-14 at 17 48 00

**Comment by DominicHolmes** [comment]
@01jongmin

**Comment by DominicHolmes** [comment]
TODO: fix googleserviceinfo.plist changes (something broke w references)

**Comment by DominicHolmes** [comment]
there are 29 pbxproj conflicts o.O

might try to merge those with an automatic script

File: PennMobile/Dining/Controllers/Statistics/DiningInsightsViewController.swift [hunk 1]
```diff
@@ -30,17 +30,21 @@ class DiningInsightsViewController: GenericViewController {
...
+            case .failure(.authenticationError):
+                //TODO: handle authenticationError
+                print("reauthentication?")
+            case .failure(.noInternet):
+                DispatchQueue.main.async { self.navigationVC?.addStatusBar(text: .noInternet) }
+            default:
+                DispatchQueue.main.async { self.navigationVC?.addStatusBar(text: .apiError) }
             }
+            
+            self.diningInsights = DiningAPI.instance.getCachedDiningInsights()
```
Comment (DominicHolmes) [inline PennMobile/Dining/Controllers/Statistics/DiningInsightsViewController.swift]: can we do this only when there is a failure? it's a little confusing that we do this all the time (even though it works). we should use the result if it was .success.

Comment (01jongmin) [reply PennMobile/Dining/Controllers/Statistics/DiningInsightsViewController.swift]: I thought that the cache exist such that even if a user, for example, opens their dining insights tab without the internet, if they had access to the internet and cached the result of their dining insights previously, they would still be able to view it.

Comment (DominicHolmes) [reply PennMobile/Dining/Controllers/Statistics/DiningInsightsViewController.swift]: resolved

**Pull request closed without merging**